jobs/src/main/scala/dcos/metronome/scheduler/impl/ReconciliationActor.scala

```diff
@@ -54,7 +56,7 @@ class ReconciliationActor(
 
   onTransition {
     case _ -> Idle =>
-      // TODO: eventBus.publish(ReconciliationFinished)
+      context.system.eventStream.publish(ReconciliationFinished())
```

Comment:
I know this is not used anymore but I guess there is no harm in keeping it already in place, right?

Reply:
looks good :)